### DISCUSSION TITLE
Several MFKZQ01LM

### DISCUSSION THREAD

**Discussion by Lahorde** [Koenkk/zigbee2mqtt#6622] [Devices]
Hello,

I'm using Aqara Cube set [MFKZQ01LM](https://www.zigbee2mqtt.io/devices/MFKZQ01LM.html) with zigbee2mqtt. It worked like a charm : with my first cube I was able to my devices in my home. I was very satisfied. So I bought 3 more. 

For these new cubes it was a pain to add it to the network. I had to do lot of attempts pressing pairing button randomly. At the end I was able to add it to network.
The problem I have is that after some time the new cubes stops working. I do not receive any events form them anymore. I press pairing button, I'm trying to add it again to network : it randomly works. I checked battery level, RSSI. All is ok.

I cannot understand why I have 1 cube that perfectly works and the other that have lot of issues.
I'm wondering if :
* the new cubes I bought have a wrong firmware
* something prevents multiple cube to communicate

Maybe it's just a zigbee2mqtt issue... I don't know, if you have some suggestions, I'm interested in!

**Comment by semaneon** [comment]
Have you found a solution to this problem?
